cmd/ltctl/report.go

```diff
@@ -105,6 +107,49 @@ func RunGenerateReportCmdF(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func generateDashboard(reports []report.Report) error {
+	dashboardFile, err := os.Create("dashboard.json")
+	if err != nil {
+		return fmt.Errorf("failed to create output file: %w", err)
+	}
+	defer dashboardFile.Close()
+
+	baseReport := reports[0]
+	newReport := reports[1]
```

Comment:
Yes I kept it simple because that's what our most common and current use-case is but you are right we should at least check and return a warning/error if there are more than two reports.